en/0x10-V1-Inventory.md

```diff
@@ -17,13 +17,13 @@ transparency within the organization, promotes software standardization and reus
 
 | # | Description | L1 | L2 | L3 |
 | :---: | :--- | :---: | :---: | :---: |
-| **1.1** | All components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
-| **1.2** | Package managers are used to manage all third-party binary components | ✓ | ✓ | ✓ |
+| **1.1** | All constituent components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
+| **1.2** | Package managers are used to manage all third-party components | ✓ | ✓ | ✓ |
 | **1.3** | An accurate inventory of all third-party components is available in a machine-readable format | ✓ | ✓ | ✓ |
 | **1.4** | Software bill-of-materials are generated for publicly or commercially available applications | ✓ | ✓ | ✓ |
-| **1.5** | Software bill-of-materials are required for new procurements | | ✓ | ✓ |
-| **1.6** | Software bill-of-materials are continuously maintained for all systems | | | ✓ |
-| **1.7** | Uniquely identify all components in a consistent, machine-readable format | ✓ | ✓ | ✓ |
+| **1.5** | Software bill-of-materials are required for new components | | ✓ | ✓ |
```

Comment:
The spec specifically calls out procurements. We need to keep this in place.

Reply:
This implies any procurement, not just for components. It is also referencing the process of acquiring the component, rather than what you actually procure. That is to say, a "procurement" is the process by which you gain something, not what you actually gained.

If you really want the word "procurement", then what you want is "procured components". Nevertheless, I have reverted.